include/irods/private/storage_tiering/proxy_connection.hpp

```diff
@@ -9,30 +9,57 @@ namespace irods {
         rErrMsg_t err_msg;
         rcComm_t* conn;
 
-        auto make(const std::string clientUser = "", const std::string clientZone = "") -> rcComm_t*
+        // Makes a proxy connection where the client is specified by the username in the parameters, and the proxy user
+        // is the service account rodsadmin for the local server.
+        auto make(const std::string clientUser, const std::string clientZone) -> rcComm_t*
         {
             rodsEnv env{};
             _getRodsEnv(env);
 
-            conn = _rcConnect(
-                       env.rodsHost,
-                       env.rodsPort,
-                       env.rodsUserName,
-                       env.rodsZone,
-                       !clientUser.empty() ?
-                           clientUser.c_str() :
-                           env.rodsUserName,
-                       !clientZone.empty() ?
-                           clientZone.c_str() :
-                           env.rodsZone,
-                       &err_msg,
-                       0, 0);
+            conn = _rcConnect(env.rodsHost,
+                              env.rodsPort,
+                              env.rodsUserName,
+                              env.rodsZone,
+                              clientUser.c_str(),
+                              clientZone.c_str(),
+                              &err_msg,
+                              0,
+                              0);
 
             clientLogin(conn);
 
             return conn;
         } // make
 
+        // Makes a proxy connection where both the proxy and client users are the service account rodsadmin for the
+        // local server.
+        auto make() -> RcComm*
+        {
+            rodsEnv env{};
+            _getRodsEnv(env);
+
+            conn = _rcConnect(env.rodsHost,
+                              env.rodsPort,
+                              env.rodsUserName,
+                              env.rodsZone,
+                              env.rodsUserName,
+                              env.rodsZone,
+                              &err_msg,
+                              0,
+                              0);
+
+            clientLogin(conn);
```

Comment:
Please open an issue to add error checking around establishing connections.

_To be handled in a separate PR._

Reply:
Opened https://github.com/irods/irods_capability_storage_tiering/issues/296 and included TODO comments.